recipes/itk-dev/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "4.11.0" %}
+
+package:
+  name: itk-dev
+  version: "{{version}}"
+
+source:
+  git_url: git://itk.org/ITK.git
+  git_tag:  v{{version}}
```

Comment:
We should be able to use the one auto generated from reliable Github, as opposed the manually created one on SourceForge.